StRoot/StFwdTrackMaker/include/Tracker/FwdHit.h

```diff
@@ -21,7 +23,7 @@ class FwdSystem : public KiTrack::ISectorSystem {
     static const int sNFttLayers = 4;
     static const int sNFstLayers = 3;
     FwdSystem(const int ndisks = FwdSystem::sNFwdLayers) : KiTrack::ISectorSystem(), mNDisks(ndisks){};
-    ~FwdSystem(){/* */};
+    ~FwdSystem() = default;
```

Comment:
Passing all tests suggests that this works with the current compilers. Interesting. When I tried to use it last time compilations failed.